workshop2/xdp_runner.go

```diff
@@ -15,39 +15,27 @@ import (
 )
 
 /*
-#cgo CFLAGS: -I/usr/include/bcc/compat
-#cgo LDFLAGS: -lbcc
-#include <bcc/bcc_common.h>
 #include <bcc/libbpf.h>
-void perf_reader_free(void *ptr);
 */
 import "C"
 
-// protocols contains minimal mappings between internet protocol
-// names and numbers for platforms that don't have a complete list of
-// protocol numbers.
-//
-// See https://www.iana.org/assignments/protocol-numbers
-//
-// On Unix, this map is augmented by readProtocols via lookupProtocol.
+const (
+	numberOfArguments  = 2
+	bpfDefaultLogLevel = 1
+	bpfLogSize         = 65536
+)
 
+// protocols is a mapping between a protocol number to its string representation
 var protocols = map[uint32]string{
-	0:  "HOPOPT",
```

Comment:
you removed it from here but the example image does have it...

Reply:
Nice! took a new screenshot